src/components/button.jsx

```diff
@@ -8,12 +8,13 @@ const Button = (props) => {
       type={props.type ?? "button"}
       disabled={props.disabled}
       class={`${classNames(styles.btn, {
-        [styles.primary]: props.category === "primary",
-        [styles.secondrary]: props.category === "secondrary",
-        [styles.tertiary]: props.category === "tertiary",
-        [styles.tertiary_light]: props.category === "teriary_light",
-        [styles.flat]: props.category === "flat",
-        [styles.trade]: props.category === "trade",
+        [styles.primary]: props.type === "primary",
+        [styles.secondrary]: props.type === "secondrary",
+        [styles.tertiary]: props.type === "tertiary",
+        [styles.tertiary_light]: props.type === "teriary_light",
+        [styles.flat]: props.type === "flat",
+        [styles.trade]: props.type === "trade",
+        [styles.reset]: props.type === "reset",
```

Comment:
If u want to apply reset style and do noyt want to use category, please use pops.type for reset. For others please change to props.category